src/App.spec.ts

```diff
@@ -509,26 +509,26 @@ describe('App', () => {
                 team: {},
               },
             },
-            { // IncomingEventType.Action (app.action)
-              ...baseEvent,
-              body: {
-                type: 'message_action',
-                callback_id: 'message_action_callback_id',
-                channel: {},
-                user: {},
-                team: {},
-              },
-            },
-            { // IncomingEventType.Action (app.action)
-              ...baseEvent,
-              body: {
-                type: 'message_action',
-                callback_id: 'another_message_action_callback_id',
-                channel: {},
-                user: {},
-                team: {},
-              },
-            },
+            // { // IncomingEventType.Action (app.action)
+            //   ...baseEvent,
+            //   body: {
+            //     type: 'message_action',
+            //     callback_id: 'message_action_callback_id',
+            //     channel: {},
+            //     user: {},
+            //     team: {},
+            //   },
+            // },
+            // { // IncomingEventType.Action (app.action)
+            //   ...baseEvent,
+            //   body: {
+            //     type: 'message_action',
+            //     callback_id: 'another_message_action_callback_id',
+            //     channel: {},
+            //     user: {},
+            //     team: {},
+            //   },
+            // },
```

Comment:
these events are still valid and should likely remain in the tests. it's just that the comment should be adjusted to describe this payload as a shortcut, we should add a global shortcut receiver event, and we should probably reorder them to keep shortcuts separated from the actions.


```suggestion
            { // IncomingEventType.Shortcut (app.shortcut)
              ...baseEvent,
              body: {
                type: 'message_action',
                callback_id: 'message_action_callback_id',
                channel: {},
                user: {},
                team: {},
              },
            },
            { // IncomingEventType.Shortcut (app.shortcut)
              ...baseEvent,
              body: {
                type: 'message_action',
                callback_id: 'another_message_action_callback_id',
                channel: {},
                user: {},
                team: {},
              },
            },
            // TODO: add global shortcut event
```